tests/performance/jit_aggregate_functions.xml

```diff
@@ -119,7 +122,7 @@
         SELECT
             {function}(value_1),
             {function}(value_2),
-            groupBitAnd(value_3),
+            sum(toUInt256(value_3)),
```

Comment:
Ok but I did not get this change.

Reply:
Is it because we need to add some function that cannot be compiled?

Reply:
@alexey-milovidov yes, and most of the functions can be compiled later, so best way is to choose aggregate function with non compilable argument.